conformance/lang/expressions/shift-expression/left_shift.balt

```diff
@@ -0,0 +1,5522 @@
+    //        = In decimal: (-9223372036854775808)
+
+    io:println(a << f); // @output -1
+    // Explanation:
+    // Value to be shifted = Equivalent binary representation : 11111111 11111111 11111111 11111111 11111111 11111111 11111111 11111111
+    // Shift amount = Before applying mask, In binary: 11111111 11111111 11111111 11111111 11111111 11111111 11111111 11000000, In decimal: -64
+    //                After applying mask, In binary: 000000, In decimal: 0
+    // Result = In binary: 11111111 11111111 11111111 11111111 11111111 11111111 11111111 11111111
+    //        = In decimal: (-1)
+
+    io:println(a << a); // @output -9223372036854775808
+    // Explanation:
+    // Value to be shifted = Equivalent binary representation : 11111111 11111111 11111111 11111111 11111111 11111111 11111111 11111111
+    // Shift amount = Before applying mask, In binary: 11111111 11111111 11111111 11111111 11111111 11111111 11111111 11111111, In decimal: -1
+    //                After applying mask, In binary: 111111, In decimal: 63
+    // Result = In binary: 10000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
+    //        = In decimal: (-9223372036854775808)
+
+    io:println(e << c); // @output -65
+    // Explanation:
+    // Value to be shifted = Equivalent binary representation : 11111111 11111111 11111111 11111111 11111111 11111111 11111111 10111111
+    // Shift amount = Before applying mask, In binary: 00000000 00000000 00000000 00000000 00000000 00000000 00000000 01000000, In decimal: 64
+    //                After applying mask, In binary: 000000, In decimal: 0
+    // Result = In binary: 11111111 11111111 11111111 11111111 11111111 11111111 11111111 10111111
+    //        = In decimal: (-65)
+}
+
+Test-Case: error
+Description: Test resultant type of the '<<' expression with 'int' operands.
+Labels: decimal, float, int, shift-expr, unary-minus, var
+
+function errorFunction() {
+    int a = 128;
+    int b = 7;
+    int c = -120;
+    int d = -9223372036854775794;
+
+    var x1 = a << b;
+    decimal _ = x1; // @error expected 'decimal' but found 'int'
+    var x2 = a << d;
+    float _ = x2; // @error expected 'float' but found 'int'
+    var x3 = c << d;
+    decimal _ = x3; // @error expected 'decimal' but found 'int'
+    var x4 = c << b;
+    float _ = x4; // @error expected 'float' but found 'int'
+
+    var x5 = 511 << 8;
+    float _ = x5; // @error expected 'float' but found 'int'
+    var x6 = -197 << -9223372036854775807;
+    float _ = x6; // @error expected 'float' but found 'int'
+}
+
+Test-Case: error
+Description: Test resultant type of the '<<' expression with 'int' as the LHS operand and builtin subtypes of integer
+             as the RHS operand.
+Labels: decimal, float, int, int:Signed8, int:Signed16, int:Signed32, int:Unsigned8, int:Unsigned16, int:Unsigned32,
+        shift-expr, unary-minus, var
+
+function errorFunction() {
+    int a = 128;
```

Comment:
Instead of creating the variables, since this is a error case, use parameters with the needed types.
eg:
```ballerina
function errorFunction(int a, int:Signed8 b....)
```

Reply:
Change in other places as well

Reply:
Addressed in https://github.com/ballerina-platform/ballerina-spec/pull/953/commits/821f1e4aa1ff3f6efe20c7dfccb163235e1b77e7